packages/flutter/lib/src/widgets/dismissible.dart

```diff
@@ -142,6 +145,9 @@ class Dismissible extends StatefulWidget {
   /// it is positive or negative.
   final double crossAxisEndOffset;
 
+  /// {@macro flutter.gestures.recognizer.dragStartBehavior}
+  final DragStartBehavior startBehavior;
```

Comment:
why is this one startBehavior instead of dragStartBehavior? (same applies to some others) (it's fine if there's a reason, i just couldn't figure it out)

Reply:
So this was an accident. For GestureDetector though, the name has to be different than dragStartBehavior or it doesn't get passed through to the initializer correctly.

Reply:
That doesn't make sense to me, can you elaborate? What is the problem?